src/serializer/MapFishPrintV3OSMSerializer.ts

```diff
@@ -1,51 +1,28 @@
-import OlSource from 'ol/source/Source';
-import OlSourceOSM from 'ol/source/OSM';
 import OlLayer from 'ol/layer/Layer';
+import OlSource from 'ol/source/Source';
+import olSourceOSM from 'ol/source/OSM';
+import MapFishPrintV3XYZSerializer from './MapFishPrintV3XYZSerializer';
 
-import BaseSerializer from './BaseSerializer';
-
-export class MapFishPrintV3OSMSerializer implements BaseSerializer {
-
-  /**
-   * The WMS layer type identificator.
-   */
-  static TYPE_OSM: string = 'osm';
+export class MapFishPrintV3OSMSerializer extends MapFishPrintV3XYZSerializer {
 
-  validateSource(source: OlSource): source is OlSourceOSM {
-    return source instanceof OlSourceOSM;
+  validateSource(source: OlSource): source is olSourceOSM {
+    return source instanceof olSourceOSM;
```

Comment:
```suggestion
  validateSource(source: OlSource): source is OlSourceOSM {
    return source instanceof OlSourceOSM;
```

Reply:
done in: [5fb9911](https://github.com/terrestris/mapfish-print-manager/pull/482/commits/5fb9911fc3f7f58244edf255f35fb900da41d7bf)